### PR TITLE
Sli documentation

### PR DESCRIPTION
**Pull request by radkomateusz**

### CODE REVIEW

File: SLI_README.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
+# SLI (Service Level Indicators)
+
+
+### Number of tables which have't have backups after X days since last change.
```
Comment (przemyslaw-jasinski) [inline SLI_README.md]: You missed 'n' in haven't and I think we should use more official return like 'have not'.

File: SLI_README.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
+# SLI (Service Level Indicators)
+
+
+### Number of tables which have't have backups after X days since last change.  
+
+This metric is measuring number of tables which are not backed up since X days from last observed change. 
+
+To measure it, it is required to have [GCP Census](https://github.com/ocadotechnology/gcp-census) application scanning all data which should be backed up 
+and also turning on daily datastore exports.
```
Comment (przemyslaw-jasinski) [inline SLI_README.md]: Link to [cloud datastore export](https://github.com/ocadotechnology/bbq/blob/master/SETUP.md#cloud-datastore-export), please

File: SLI_README.md (hunk 1 above)
Comment (radkomateusz) [reply SLI_README.md]: done

File: SLI_README.md (hunk 2 above)
Comment (radkomateusz) [reply SLI_README.md]: done

**Pull request merged**